modules/update-service-install-web-console.adoc

```diff
@@ -0,0 +1,34 @@
+[id="update-service-install-web-console_{context}"]
+= Installing the {product-title} Update Service by using the web console
+
+You can use the {product-title} web console to install the {product-title} Update Service Operator.
+
+.Procedure
+
+To install the {product-title} Update Service Operator using the {product-title} web console:
+
+. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+. Choose *{product-title} Update Service* from the list of available Operators, and click *Install*.
+
+. FIXME: Does this apply? Ensure that the *All namespaces on the cluster* is selected under *Installation Mode*.
+
+. Select a namespace for *Installed Namespace*, such as the default FIXME-default-namespace.
+
+. FIXME: Does this apply? Select *Enable operator recommended cluster monitoring on this namespace*.
```

Comment:
Does this apply?

Reply:
I'm [still not sure][1] :(

[1]: https://github.com/openshift/openshift-docs/pull/26219#discussion_r570750641